src/routes/popup/swap/utils/alarms/swap-monitor/swap-fee-processor.ts

```diff
@@ -62,6 +70,12 @@ export async function processWanderFee(swap: SwapData): Promise<boolean> {
       return true;
     }
 
+    const isAlreadyProcessed = await checkIfWanderFeeAlreadyProcessed(swap);
```

Comment:
Double fee sending should not happen. But added this check to prevent that if it happens.